### ISSUE TITLE
Mobile page navigation doesn't work if the pages are not the immediate children of the `body`

### ISSUE THREAD

**Issue by VsevolodGolovanov** [primefaces/primefaces#759]
```
<h:body>
    <h:panelGroup id="my_page_container">
        <pm:page>
        ...
```

Attempting to navigate between pages (p:link, p:button or PrimeFaces.Mobile.navigate) results only in a console error

> Uncaught Error: cannot call methods on pagecontainer prior to initialization; attempted to call method 'change'

jQuery Mobile automatically correctly determines the page container to be `#my_page_container`, and that's where it creates the widget instance. But the problem is PrimeFaces.Mobile.navigate expects the page container to be on the `body` tag. Probably should use `$(":mobile-pagecontainer")` instead.

Encountered with PF5.2.12, seems to be relevant in 5.3 judging by the core.mobile.js.

**Comment by tandraschko** [comment]
Would be great if you could provide a PR.

**Comment by tandraschko** [comment]
PrimeFaces Mobile will be deprecated  in 6.2 and removed in 6.3 ( see https://www.primefaces.org/primefaces-6-2-roadmap/)
Therefor i will close this issue as won't fix. If you will provide a PR to fix this issue until 6.2 RC1, i can merge it.
